doc/user/content/integrations/cli/reference/sql.md

```diff
@@ -0,0 +1,41 @@
+---
+title: mz sql
+description: The `mz sql` command executes SQL statements in a region.
+menu:
+  main:
+    parent: cli-reference
+    weight: 1
+---
+
+The `mz sql` command executes SQL statements in a region.
+
+```shell
+mz sql [<REGION>] [-- psql options...]
```

Comment:
Ergonomically, it'd be great if I could run something like `mz shell <REGION> <CLUSTER>` and have it set my cluster on connection.

Reply:
This should be allowed in the `[-- psql options...]` placeholder now that we have https://github.com/MaterializeInc/materialize/issues/14573. But we should definitely call it out as an example

Reply:
Copy that! Yeah, I think having some syntactic sugar that allows you to type `mz shell aws/us-east-1 mz_instrospection` instead of `mz shell aws/us-east-1 -- --cluster=mz_introspection` would be more pleasant. 

On top of that, have we considered _requiring_ users to specify a cluster when they want to connect via the mz cli? May not be something we want to deal with right now, but it has some potentially beneficial UX tradeoffs I think.

Reply:
It would be wonky to have to have to specify default on your first connection, or if you only have one cluster. I would like to print out the selected cluster however, see below